testutil/ensurelogchecker.go

```diff
@@ -124,3 +126,45 @@ func CheckEnsureLoopLogging(filename string, c *check.C, allEnsuresMustContainTr
 		}
 	}
 }
+
+// within a given folder, finds all files that contain an
+// implementation of the StateManager interface
+func GetListOfStateManagerImplementers(folder string) ([]string, error) {
```

Comment:
s/GetList//

Reply:
Did you mean to say you want to remove `ListOf` instead of `GetList`? So this:
```suggestion
func GetStateManagerImplementers(folder string) ([]string, error) {
```
?

Reply:
sorry, I missed the "Of" bit, I meant also to remove the "Get", in generally golang naming conventions recommend avoiding Get in function names. We sometimes use it anyway to mean "it's sort of time consuming" or in non-exported functions but it's not clear it adds something here